topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md

```diff
@@ -0,0 +1,1244 @@
+
+
+# Analysis Strategy
+
+Most scRNA sequencing techniques use pooled-sequencing approaches to generate a higher throughput of data by performing amplification and sequencing upon multiple cells in the same "pool". From a bioinformatics standpoint, this means that the output FASTQ data from the sequencer is batch-specific and contains all the sequences from multiple cells.
+
+In this tutorial, we will perform pre-processing upon single-cell RNA FASTQ batch data to generate an *N*-by-*M*  count matrix of *N* cells and *M* genes, with each element indicating the level of expression of that gene in a particular cell. This matrix can be used in further downstream analyses, which will be covered in another tutorial. 
+
+The first part of this tutorial will use example *FASTQ* Batch data, which we will pair into a Galaxy *Collection* and perform barcode extraction and annotation upon. Alignment and quality control will also be performed, and we will see how to construct a rudimentary count matrix. 
+
+The second part of this tutorial will deal with multiple batches, and a different set of example count matrices will be used in which to merge and perform quality control upon. This will produce a final count matrix valid for downstream analysis.
+
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Single Batch Processing
+
+### Data upload and organization
+
+The size of scRNA FASTQ files are typically in the gigabyte range and are somewhat impractical for training purposes, so we will expediate the analysis by using a smaller subset of actual batch data. This data is available at [`Zenodo`](https://zenodo.org/record/1345635) where the example FASTQ paired batch files are hosted, as well as a barcodes file and GTF file for the *Mus Musculus* genome version mm10.
+
+The batch files are the first of the set of multiple batches, and originate from the first sequencing plate. We will explain this more in detail later, but for now we should make note of this when renaming our files in step 6 of the hands-on below.
+
+> ### {% icon hands_on %} Hands-on: Data upload and organization
+>
+> 1. Create a new history and name it something meaningful (*e.g.* scRNA-seq single batch tutorial)
+> 1. Open the Data Upload Manager by selecting *Get Data* from the Tool Panel and clicking *Upload File*
+> 1. Select *Paste/Fetch Data*
+> 1. Copy each link for the reads (`SRR5683689_1.fastq.gz`, `SRR5683689_1.fastq.gz`), annotation (`Mus_musculus.GRCm38.93.mm10.UCSC.ncbiRefSeq.gtf`), and barcode (`celseq_barcodes.192.tabular`) files, and paste each link into a separate text field
+>    - Set the datatype of the read (.fastq.gz) files to **fastq.gz**
+>    - Set the datatype of the annotation (.gtf) file to **tabular** and assign the Genome as **mm10**
+> 1. Click *Start*
```

Comment:
You have to say here, they need to rename their files in Galaxy because Galaxy takes the links as names. 

Furthermore, the Zenodo links appear red. It is not working to upload them